xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -100,10 +97,7 @@
 #define ADDON_INSTANCE_VERSION_PVR                    "6.1.0"
 #define ADDON_INSTANCE_VERSION_PVR_MIN                "6.1.0"
```

Comment:
increase both version defines to 7.0.0.

Reply:
I can increase now, but has leaved as long the WIP is set, see my note above **"Therefore, not even a version change is introduced, this comes in the end when everything is ready so far."** :smirk: (with a version re add at last commit, I want to prevent this all the times until end)